NeptunesPrideStateDownloader/Program.cs

```diff
@@ -103,7 +103,7 @@ private static void WriteDone()
                 if (!File.Exists(path))
                 {
                     Console.WriteLine($"Found new tick: {tick}, saving state.");
-                    File.WriteAllText(path, json);
+                    File.WriteAllText(path, JsonConvert.SerializeObject(json, Formatting.Indented));
```

Comment:
`json` is the deserialized JSON string, so serializing it won't format it, it'll just escape all the quotes and wrap it in extra quotes.  I think serializing `state` should work here.